Core/clim-core/gadgets.lisp

```diff
@@ -1646,15 +1654,18 @@ and must never be nil.")
 ;; slider's sheet, probably his child).
 ;; ----------------------------------------------------------
 
+(defgeneric convert-position-to-value (slider-pane position))
+(defgeneric convert-value-to-position (slider-pane))
```

Comment:
this could use a documentation string or at least a comment describing the purpose and the contract.

Reply:
also parameter name is misleading, because I've noticed that you use argument `position` as `event`. maybe it is worth changing?